File: Java/23_Math/LC_2578_SplitWithMinimumSum.java
class Solution {
    public int splitNum(int num) {
        char[] cs = Integer.toString(num).toCharArray();
        Arrays.sort(cs);
        int[] a = new int[2];
        for (int i = 0; i < cs.length; i++) {
            a[i % 2] = a[i % 2] * 10 + cs[i] - '0';
        }
        return a[0] + a[1];
    }
}

public class LC_2578_SplitWithMinimumSum {

}

File: Swift/Leetcode.playground/Sources/23_Math/LC_2578_SplitWithMinimumSum.swift
class Solution {
    func splitNum(_ num: Int) -> Int {
        let cs = Array(String(num))
        let sortedCs = cs.sorted()
        var a = [Int](repeating: 0, count: 2)
        for i in 0..<sortedCs.count {
            a[i % 2] = a[i % 2] * 10 + Int(String(sortedCs[i]))!
        }
        return a[0] + a[1]
    }
}